dashboard/app/controllers/api/v1/peer_review_submissions_controller.rb

```diff
@@ -32,4 +35,63 @@ def index
 
     render json: submissions.values
   end
+
+  def report
```

Comment:
nit: since this action is always sending a csv attachment, indicate that in the name (e.g. report_csv)

Existing actions that use the `CsvDownload` branch on the request format and only use it when csv is requested. [Example](https://github.com/code-dot-org/code-dot-org/blob/a53a80cc3f23726a4c4fcf924f0dc7bb0ae6f16e/dashboard/app/controllers/api/v1/pd/workshop_organizer_survey_report_controller.rb#L20-L28)